src/components/molecules/widgets/InjuredAccidentsWithPedestrians.tsx

```diff
@@ -26,20 +26,30 @@ const useStyle = makeStyles((theme: Theme) => ({
 
 const InjuredAccidentsWithPedestrians: FC<IProps> = ({ data, segmentText }) => {
   const classes = useStyle();
-  const { items } = data;
+  const  items  = [
```

Comment:
Why did you replace dynamic data with a static mock?

Reply:
will be like that until I finish the features so I can test the stacked bar chart. won't be in the final pull request.